<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Register</class>
 <widget class="QWidget" name="Register">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>700</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Posefect - Register</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*  {
	background-color: rgb(255, 243, 224);
	font: 57 10pt &quot;Montserrat Medium&quot;;
}
QGroupBox {
	font-size: 16px;
}
#mail_label, #pass_label {
	padding: 10px 5px;
}
QLineEdit, QDateEdit, QComboBox {
	border-bottom: 2px solid #c17900;
	border-radius: 0;
	padding: 10px 5px;
	cursor: pointer;
}
QPushButton {
	margin-top: 15px;
	background-color: rgb(249, 168, 37);
	padding: 12px;
	border-radius: 20px;
}
QCommandLinkButton {
	background-color: transparent;
	text-align: center;
}</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>751</width>
     <height>551</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/imagePath/register.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="textFormat">
       <enum>Qt::RichText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Signup</string>
      </property>
      <widget class="QLabel" name="login_heading_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>351</width>
         <height>44</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Montserrat ExtraBold</family>
         <pointsize>14</pointsize>
         <weight>10</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 81 14pt &quot;Montserrat ExtraBold&quot;;</string>
       </property>
       <property name="text">
        <string>Get Started to Become Perfect!</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_5">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>175</y>
         <width>261</width>
         <height>42</height>
        </rect>
       </property>
       <property name="maxLength">
        <number>1000</number>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="cursorPosition">
        <number>0</number>
       </property>
       <property name="placeholderText">
        <string>Enter your password</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QCommandLinkButton" name="CreateAnAcc_3">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>470</y>
         <width>111</width>
         <height>40</height>
        </rect>
       </property>
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="acceptDrops">
        <bool>true</bool>
       </property>
       <property name="toolTipDuration">
        <number>300</number>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhTime</set>
       </property>
       <property name="text">
        <string>Login Now</string>
       </property>
       <property name="icon">
        <iconset resource="../icons/icons.qrc">
         <normaloff>:/iconPrefix/move-right.png</normaloff>:/iconPrefix/move-right.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
      <widget class="QLabel" name="mail_label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Full Name</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>203</x>
         <y>396</y>
         <width>161</width>
         <height>61</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="acceptDrops">
        <bool>true</bool>
       </property>
       <property name="toolTipDuration">
        <number>300</number>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Register</string>
       </property>
       <property name="icon">
        <iconset resource="../icons/icons.qrc">
         <normaloff>:/iconPrefix/add-user.png</normaloff>:/iconPrefix/add-user.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="autoRepeat">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_6">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>78</y>
         <width>261</width>
         <height>42</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="maxLength">
        <number>1000</number>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="cursorPosition">
        <number>0</number>
       </property>
       <property name="placeholderText">
        <string>e.g. John Doe</string>
       </property>
       <property name="cursorMoveStyle">
        <enum>Qt::VisualMoveStyle</enum>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="pass_label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>180</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Password</string>
       </property>
      </widget>
      <widget class="QLabel" name="mail_label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>61</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Email</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_7">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>128</y>
         <width>261</width>
         <height>42</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="maxLength">
        <number>1000</number>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="cursorPosition">
        <number>0</number>
       </property>
       <property name="placeholderText">
        <string>e.g. somemail@example.com</string>
       </property>
       <property name="cursorMoveStyle">
        <enum>Qt::VisualMoveStyle</enum>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QDateEdit" name="dateEdit">
       <property name="geometry">
        <rect>
         <x>99</x>
         <y>271</y>
         <width>261</width>
         <height>41</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="pass_label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>280</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>D.O.B</string>
       </property>
      </widget>
      <widget class="QLabel" name="pass_label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>320</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Role</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>320</y>
         <width>261</width>
         <height>31</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="editable">
        <bool>false</bool>
       </property>
       <property name="currentText">
        <string/>
       </property>
       <property name="currentIndex">
        <number>-1</number>
       </property>
       <property name="maxVisibleItems">
        <number>3</number>F
       </property>
       <property name="minimumContentsLength">
        <number>2</number>
       </property>
       <property name="iconSize">
        <size>
         <width>5</width>
         <height>5</height>
        </size>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLabel" name="pass_label_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>230</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Confirm Pass</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_8">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>225</y>
         <width>261</width>
         <height>42</height>
        </rect>
       </property>
       <property name="maxLength">
        <number>1000</number>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="cursorPosition">
        <number>0</number>
       </property>
       <property name="placeholderText">
        <string>Enter your password</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
      <zorder>pass_label_3</zorder>
      <zorder>login_heading_3</zorder>
      <zorder>lineEdit_5</zorder>
      <zorder>CreateAnAcc_3</zorder>
      <zorder>mail_label_3</zorder>
      <zorder>pushButton_3</zorder>
      <zorder>lineEdit_6</zorder>
      <zorder>mail_label_4</zorder>
      <zorder>lineEdit_7</zorder>
      <zorder>pass_label_4</zorder>
      <zorder>dateEdit</zorder>
      <zorder>pass_label_5</zorder>
      <zorder>comboBox</zorder>
      <zorder>pass_label_6</zorder>
      <zorder>lineEdit_8</zorder>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>751</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Montserrat ExtraBold</family>
        <pointsize>28</pointsize>
        <weight>10</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 81 28pt &quot;Montserrat ExtraBold&quot;;</string>
      </property>
      <property name="text">
       <string>POSEFECT</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="indent">
       <number>0</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../icons/icons.qrc"/>
  <include location="../font/fonts.qrc"/>
  <include location="../Images/images.qrc"/>
 </resources>
 <connections/>
</ui>
